src/doc/src/reference/profiles.md

```diff
@@ -168,14 +168,14 @@ of longer linking time.
 
 The valid options are:
```

Comment:
Just FYI, [rustc](https://doc.rust-lang.org/nightly/rustc/codegen-options/index.html#lto) doesn't sort them by optimization levels. And perhaps when ehuss wrote this was thinking of putting boolean options closer to each other. That also makes sense.

I think the reorder is fine, though should we mention the order as well? Like

```suggestion
The valid options from most to least optimizing are:
```

Reply:
Yeah that would be great.

I am going to merge this now, as it is not hard to revert if needed. Thank you